pkg/install/resources.go

```diff
@@ -43,10 +43,10 @@ var (
 	DefaultVeleroPodMemRequest    = "128Mi"
 	DefaultVeleroPodCPULimit      = "1000m"
 	DefaultVeleroPodMemLimit      = "512Mi"
-	DefaultNodeAgentPodCPURequest = "500m"
-	DefaultNodeAgentPodMemRequest = "512Mi"
-	DefaultNodeAgentPodCPULimit   = "1000m"
-	DefaultNodeAgentPodMemLimit   = "1Gi"
+	DefaultNodeAgentPodCPURequest = "0"
```

Comment:
let's add a comment to explain this is for "best effort" QoS

Reply:
Done